card/mod/03_machines/set/type/css.rb

```diff
@@ -38,7 +38,7 @@ def get_inclusion_defaults nested_card
   view :core do |args|
     # FIXME: scan must happen before process for inclusion interactions to work, but this will likely cause
     # problems with including other css?
-    process_content ::CodeRay.scan( _render_raw, :css ).div, :size=>:icon
+    process_content ::CodeRay.scan( _render_raw, :css ).div, :content_opts=>{:size=>:icon}
```

Comment:
Line is too long. [90/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.<br>Space inside parentheses detected.